src/network/C4Network2UPnP.h

```diff
@@ -21,7 +21,7 @@
 
 class C4Network2UPnP
 {
-	struct C4Network2UPnPP *p;
+	class C4Network2UPnPP *p;
```

Comment:
If you change this you need to change src/network/C4Network2UPnPWin32.cpp too to avoid warnings (and potential link-time issues with MSVC because it uses mangling that breaks this)